dali/test/python/reader/test_fits.py

```diff
@@ -0,0 +1,161 @@
+# Copyright (c) 2020-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nvidia.dali import Pipeline
+import nvidia.dali.fn as fn
+import os
+from astropy.io import fits
+import numpy as np
+import tempfile
+import random
+from test_utils import to_array
+from numpy.testing import assert_array_equal
+
+rng = np.random.RandomState(12345)
+
+gds_data_root = '/scratch/'
+if not os.path.isdir(gds_data_root):
+    gds_data_root = os.getcwd() + "/scratch/"
+    if not os.path.isdir(gds_data_root):
+        os.mkdir(gds_data_root)
+        assert os.path.isdir(gds_data_root)
```

Comment:
You don't need that for fits. It is used for the GPU Direct Storage solution that works only on the real file system, as our tests run inside the docker environment and its tmp file system doesn't work with GDS so we need to mount the external file system and use it for tests.

Reply:
Sure, addressed by removing this fragment.